### PR TITLE
INTYGFV-14404: Changed already existing messages to INFO response.

### PR DESCRIPTION
**Pull request by karlssone**

### CODE REVIEW

File: web/src/main/java/se/inera/intyg/webcert/web/integration/interactions/sendmessagetocare/SendMessageToCareResponderImpl.java [hunk 1]
```diff
@@ -51,19 +51,26 @@ public SendMessageToCareResponseType sendMessageToCare(String logicalAddress, Se
             arendeService.processIncomingMessage(ArendeConverter.convert(request));
             result.setResultCode(ResultCodeType.OK);
         } catch (WebCertServiceException e) {
-            result.setResultCode(ResultCodeType.ERROR);
             switch (e.getErrorCode()) {
+                case MESSAGE_ALREADY_EXISTS:
+                    result.setResultCode(ResultCodeType.INFO);
```
Comment (karlssone) [inline web/src/main/java/se/inera/intyg/webcert/web/integration/interactions/sendmessagetocare/SendMessageToCareResponderImpl.java:56]: Should I set setErrorId?

Comment (karlssone) [reply web/src/main/java/se/inera/intyg/webcert/web/integration/interactions/sendmessagetocare/SendMessageToCareResponderImpl.java:56]: This is the response that will be sent to FK. Is this ok?

**Comment by karlssone** [comment]
Should I make any changes in sendMessageToRecipient?

File: web/src/main/java/se/inera/intyg/webcert/web/integration/interactions/sendmessagetocare/SendMessageToCareResponderImpl.java [hunk 2]
```diff
@@ -51,19 +51,26 @@ public SendMessageToCareResponseType sendMessageToCare(String logicalAddress, Se
...
+                    result.setResultText(e.getMessage());
+                    LOG.info("Could not process incoming message to care. Message already exists. {} : {}", e.getErrorCode(),
```
Comment (mww-sopra-steria) [inline web/src/main/java/se/inera/intyg/webcert/web/integration/interactions/sendmessagetocare/SendMessageToCareResponderImpl.java:58]: Can we include the message id and maybe even the certificate id, to make it easier to find situations when this happens?

Same for the other logs.

**Review by mww-sopra-steria — CHANGES_REQUESTED** [review]
Would be great with some modifications to the log and the message itself that we returns.

File: web/src/main/java/se/inera/intyg/webcert/web/integration/interactions/sendmessagetocare/SendMessageToCareResponderImpl.java (hunk 1 above)
Comment (mww-sopra-steria) [reply web/src/main/java/se/inera/intyg/webcert/web/integration/interactions/sendmessagetocare/SendMessageToCareResponderImpl.java:56]: Yes!

**Review by mww-sopra-steria — APPROVED** [review]

**Pull request merged**